package.json

```diff
@@ -145,6 +145,7 @@
     "node-sass": "4.9.2",
     "nyc": "11.2.1",
     "react": "16.4.2",
+    "react-dev-utils": "5.0.1",
```

Comment:
Let's not pull in most of create-react-app just for this code: https://github.com/facebook/create-react-app/blob/next/packages/react-dev-utils/errorOverlayMiddleware.js#L9-L23

This will also allow us to drop https://github.com/zeit/next.js/blob/21996a877d2788964d348d2bdc46c9a852277fd0/server/lib/query-parser.js

Reply:
Good point. Nevertheless, the main part of the code is https://github.com/facebook/create-react-app/blob/next/packages/react-dev-utils/launchEditor.js
I am not sure if we want to maintain this file.

Reply:
I wonder how @Timer would feel about moving the editor open code into a separate package so that it can be used by consumers of react-error-overlay

Reply:
For now I used @sindresorhus [open-editor](https://github.com/sindresorhus/open-editor) package. The aim of that package is to do exactly that. Nevertheless open-editor doesn't do editor guessing. 

I created an issue to discuss it there further: https://github.com/sindresorhus/open-editor/issues/4

Reply:
We have considered splitting the editor logic into a separate package but haven't had any compelling cases to do so; if you're worried about size I'm sure we can figure out a way to accommodate this.

Dan and I are going to focus heavily on v2 in September, so we'll make a decision about this then. Maybe we can move it into the overlay package itself.

Reply:
Sounds great! 👍

Reply:
@HaNdTriX until then lets move the CRA editor open code into Next.js 👍